packages/modules/alpha_ess/bat.py

```diff
@@ -18,46 +18,52 @@ def get_default_config() -> dict:
         "name": "Alpha ESS Speicher",
         "id": 0,
         "type": "bat",
-        "configuration":
-        {
+        "configuration": {
             "version": 1
         }
     }
 
 
 class AlphaEssBat(AbstractBat):
-    def __init__(self, device_id: int, component_config: dict, tcp_client: modbus.ModbusClient) -> None:
+    def __init__(self, device_id: int, component_config: dict,
+                 tcp_client: modbus.ModbusClient) -> None:
         try:
             super().__init__(device_id, component_config, tcp_client)
         except Exception as e:
             self.process_error(e)
 
     def get_values(self) -> BatState:
-        log.MainLogger().debug("Komponente "+self.data["config"]["name"]+" auslesen.")
+        log.MainLogger().debug("Komponente " + self.data["config"]["name"] +
+                               " auslesen.")
         # keine Unterschiede zwischen den Versionen
         sdmid = 85
 
         time.sleep(0.1)
-        voltage = self.client.read_holding_registers(0x0100, modbus.ModbusDataType.INT_16, unit=sdmid)
+        voltage = self.client.read_holding_registers(
+            0x0100, modbus.ModbusDataType.INT_16, unit=sdmid)
```

Comment:
Es wundert mich etwas das in diesem merge-commit hier zu finden. Denn eigentlich war an der Stelle kein Konflikt? Nun gut, nicht so wichtig.

Generell kann man sich natürlich die Frage stellen wie lang man Zeilen maximal haben möchte. Ich habe für die Konfiguration von flake8 120 Zeichen vorgegeben. Das ist so in etwa die Zeilenlänge bei der auf einem Standard 1920x1080-Monitor noch ein side-by-side-Diff möglich ist. Vor diesem Commit war die Zeile 103 Zeichen lang, also definitiv im Limit.

Über den Geschmack der maximalen Zeilenlänge kann man natürlich vortrefflich streiten. Es wäre aber zumindest gut sich projektweit auf einen Wert festzulegen und wenn es kürzer sein soll auch die flake8-Konfig anzupassen.

Oder gab es einen anderen Grund für die vielen neuen Zeilenumbrüche?